sigopt_sklearn/search.py

```diff
@@ -227,7 +228,19 @@ def _check_bounds():
         # generate sigopt experiment parameters
         return [_transform_param(name, bounds) for (name, bounds) in param_domains.items()]
 
-    def _create_sigopt_exp(self, conn, folds):
+    def create_experiment(self, num_folds):
```

Comment:
Well it depends on things like the estimator, `sigopt_connection`, and the parameter domains. So it could be made static but those things would have to be passed as arguments.